src/fragments/lib/graphqlapi/flutter/subscribe-data.mdx

```diff
@@ -1,43 +1,65 @@
-Subscribe to mutations for creating real-time clients.  
+Subscribe to mutations for creating real-time clients.
 
 ## Setup subscription with callbacks
 
-To listen to create mutations, you can use the following code sample:
+To listen to create mutations, you can use the following code sample. When creating subscriptions, a [`Stream`](https://api.dart.dev/stable/dart-async/Stream-class.html) object will be returned to you. This `Stream` will continue producing events until either the subscription encounters an error, or you cancel the subscription. In the case of [`await for`](https://dart.dev/tutorials/language/streams), this cancellation occurs when breaking out of the loop.
```

Comment:
Nit: no comma before the "or" since the following clause is dependent 

May also be a better segue into code block if we switch the order of the two sentences.

Reply:
Good catch, thanks! Fixed 👍